api/src/test/java/org/openmrs/module/locationbasedaccess/aop/LocationUtilsTest.java

```diff
@@ -0,0 +1,190 @@
+		executeDataSet(XML_FILENAME_WITH_PERSON_DATA);
+	}
+
+	@Test
+	public void getPersonLocation_shouldReturnPersonLocation(){
+		Person person3 = personService.getPerson(DEMO_PERSON3_ID);
+		Location location1 = locationService.getLocation(DEMO_LOCATION1_ID);
+		PersonAttributeType personAttributeType = personService.getPersonAttributeType(DEMO_PERSONATTRIBUTE1_ID);
+		PersonAttribute personAttribute = new PersonAttribute(personAttributeType,location1.getUuid());
+		person3.addAttribute(personAttribute);
+		Location location3 = LocationUtils.getPersonLocation(person3);
+		assertEquals(location3,location1);
+	}
+
+	@Test
+	public void getPersonLocation_shouldReturnNullIfNoLocationAttribute() {
+		Person person2 = personService.getPerson(DEMO_PERSON2_ID);
+		PersonAttributeType personAttributeType = personService.getPersonAttributeType(DEMO_PERSONATTRIBUTE1_ID);
+		assertNull(person2.getAttribute(personAttributeType));
+		Location location = LocationUtils.getPersonLocation(person2);
+		assertNull(location);
+	}
+
+	@Test
+	public void doesPersonBelongToGivenLocationShouldBeTrue_ifPersonBelongToSameLocation(){
+		Person person2 = personService.getPerson(DEMO_PERSON2_ID);
+		PersonAttributeType personAttributeType = personService.getPersonAttributeType(DEMO_PERSONATTRIBUTE1_ID);
+		assertNull(person2.getAttribute(personAttributeType));
+		Location location1 = locationService.getLocation(DEMO_LOCATION1_ID);
+		PersonAttribute personAttribute = new PersonAttribute(personAttributeType,location1.getUuid());
+		person2.addAttribute(personAttribute);
+		String sessionLocationUuid = location1.getUuid();
+		Boolean aBoolean = LocationUtils.doesPersonBelongToGivenLocation(person2,personAttributeType,sessionLocationUuid);
+		assertTrue(aBoolean);
+	}
+
+	@Test
+	public void doesPersonBelongToGivenLocationShouldBeFalse_ifPersonNotBelongToSameLocation(){
+		Person person2 = personService.getPerson(DEMO_PERSON2_ID);
+		PersonAttributeType personAttributeType = personService.getPersonAttributeType(DEMO_PERSONATTRIBUTE1_ID);
+		assertNull(person2.getAttribute(personAttributeType));
+		String sessionLocationUuid1 = "ef93c695-ac43-450a-93f8-4b2b4d50a3c9";
+		Boolean aBoolean1 = LocationUtils.doesPersonBelongToGivenLocation(person2,personAttributeType,sessionLocationUuid1);
+		assertFalse(aBoolean1);
+		Location location1 = locationService.getLocation(DEMO_LOCATION1_ID);
+		Location location2 = locationService.getLocation(DEMO_LOCATION2_ID);
+		assertNotEquals(location1.getUuid(),location2.getUuid());
+		PersonAttribute personAttribute = new PersonAttribute(personAttributeType,location1.getUuid());
+		person2.addAttribute(personAttribute);
+		String sessionLocationUuid2 = location2.getUuid();
+		Boolean aBoolean2 = LocationUtils.doesPersonBelongToGivenLocation(person2,personAttributeType,sessionLocationUuid2);
+		assertFalse(aBoolean2);
+	}
+
+	@Test
+	public void doesUserBelongToGivenLocationShouldBeTrue_ifUserBelongToSameLocation(){
```

Comment:
it shold be changed as ```doesUserBelongToGivenLocation_shouldBeTrueIfUserBelongToSameLocation```? For other method as well

Reply:
@suthagar23  fixed can check PR now.